20240904 oc faculty en (#1167)
utelecon/utelecon.github.io: src/components/en/systems/zoom/InstallAppPc.mdx

  */}

1. Go to the Zoom "[Download Center](https://zoom.us/download)".

> 2. Please download the app that is compatible with your operating system. You may need to download from other than the blue "Download" button, such as if you are using an Apple Silicon Mac. Please pay attention to the instructions on the page.

Reviewer: ```suggestion
2. Please download the app that corresponds to your operating system. You may need to download from other than the blue "Download" button, such as if you are using an Apple Silicon Mac. Please pay attention to the instructions on the page.
```
